vncviewer/CMakeLists.txt

```diff
@@ -61,6 +61,11 @@ elseif(APPLE)
   target_link_libraries(vncviewer "-framework IOKit")
 else()
   target_link_libraries(vncviewer ${X11_Xi_LIB})
+
+  if(X11_Xrandr_LIB)
+    add_definitions(-DHAVE_XRANDR)
+    target_link_libraries(vncviewer ${X11_Xrandr_LIB})
+  endif()
```

Comment:
This seems to be in the wrong commit.

Reply:
Moved to the commit introducing tooltips.